packages/app-desktop/package.json

```diff
@@ -138,6 +138,7 @@
     "@fortawesome/fontawesome-free": "^5.13.0",
     "@joeattardi/emoji-button": "^4.6.0",
     "@joplin/lib": "~2.9",
+    "@joplin/pdf-viewer": "~1.0",
```

Comment:
Remember it should be ~2.9. Having all the packages at the same version makes it easier to make deployments.